ui/cluster-admin/src/components/ClusterListItem.tsx

```diff
@@ -46,7 +54,12 @@ const ClusterListItem: React.FC<ClusterListItemProps> = (
   }
 
   return (
-    <DataGridRow style={{ cursor: "pointer" }} onClick={() => openDetails()}>
+    <DataGridRow
+      className={`cursor-pointer ${
+        showClusterDetails === props?.cluster?.metadata?.name ? "active" : ""

```

Comment:
The `clusterDetails.cluster` should already provide the necessary data. No need to use `showClusterDetails` here.